dist/bundle.js

```diff
@@ -1,12 +1,340 @@
-!(function (e, o) {
-  "object" == typeof exports && "object" == typeof module
-    ? (module.exports = o())
-    : "function" == typeof define && define.amd
-    ? define([], o)
-    : "object" == typeof exports
-    ? (exports.Toggle = o())
-    : (e.Toggle = o());
+!(function (e, t) {
```

Comment:
@juliansg022 Could you please minify this bundle file. Thanks! 

Tip: Take a look at the `bundle.js` from the other Tools we have built.

Reply:
@juanhurtado10 I think with the correction applied the bundle file should be minified